index.html

```diff
@@ -421,8 +421,10 @@ <h2><dfn>Monetization</dfn> interface</h2>
           [Exposed=Window, SecureContext]
           interface Monetization : EventTarget {
             readonly attribute MonetizationState state;
-            attribute EventHandler onstart;
-            attribute EventHandler onprogress;
+            attribute EventHandler onmonetizationstart;
```

Comment:
these fields don't exist in the current implementation

Reply:
Yeah, and the events here aren't CustomEvent with detail